src/proposals/20230802/SparkEthereum_20230802.t.sol

```diff
@@ -303,6 +303,84 @@ contract SparkEthereum_20230802Test is SparkTestBase, TestWithExecutor {
         sparkE2eTest(POOL, makeAddr("newUser"));
     }
 
+    function testSpellExecution_liquidations() public {
+        address lp = makeAddr("lp");
+
+        address user1 = makeAddr("user1");
+        address user2 = makeAddr("user2");
+        address user3 = makeAddr("user3");
+
+        address liquidator1 = makeAddr("liquidator1");
+        address liquidator2 = makeAddr("liquidator2");
+
+        ReserveConfig[] memory configs = createConfigurationSnapshot('', POOL);
+        ReserveConfig memory dai = _findReserveConfig(configs, DAI);
+        ReserveConfig memory weth = _findReserveConfig(configs, WETH);
+
+        // Ensure WETH liquidity
+        _deposit(weth, POOL, lp, 1_000_000e18);
+
+        // Setup some positions ahead of time that can later be liquidated
+        // Position is 350 ETH so ~700k
+        _deposit(dai, POOL, user2, 1_000_000e18);
+        this._borrow(weth, POOL, user2, 350e18, false);
+
+        _deposit(dai, POOL, user3, 1_000_000e18);
+        _deposit(weth, POOL, user3, 1_000e18);
+        this._borrow(weth, POOL, user3, 1_000e18, false);
+
+        _executePayload(address(payload));
+
+        // --- Test 1 - Cannot Borrow More than small amount against DAI ---
+
+        // Refresh the configs just in case
+        configs = createConfigurationSnapshot('', POOL);
+        dai = _findReserveConfig(configs, DAI);
+        weth = _findReserveConfig(configs, WETH);
+
+        // Deposit 1m
+        _deposit(dai, POOL, user1, 1_000_000e18);
+
+        // Should only be able to borrow small amounts of ETH
+        // 1m * 1% = $10k = ~5 ETH (assume price of 2k / ETH)
+        this._borrow(weth, POOL, user1, 1e18, false);
+
+        // Cannot borrow 10
```

Comment:
Cannot borrow any more

Reply:
Went with larger amount as you can borrow a bit more.